src/commands/router.ts

```diff
@@ -89,12 +89,16 @@ export async function onInteractionReceived(interaction: Interaction) {
     await interaction.deferReply();
     await targetCommand.execute(interaction);
 
+    if (interaction.guildId === null) {
```

Comment:
oh actually, I think this will prevent users from being able to use some commands by DMing the bot, so maybe it's better to just use `unknown` for the commandMonitor if interaction.guildId is null/undefined, and let the commands decide if they need guildId or not (as they do currently)